src/components/ClassroomButton.astro

```diff
@@ -13,24 +13,34 @@ interface Props {
 }
 
 const { text, color, href } = Astro.props;
-
+let colorHover: string;
 let colorClass: string;
+/* hover colors to be changed*/
```

Comment:
it is better if you choose them now so we do not forget. ask the designers for them,